src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -143,12 +143,24 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
             throw new EntityDoesNotExistException(
                     "Trying to get feedback sessions for a course that does not exist.");
         }
+        return getFeedbackSessionsForUserInCourseSkipCheck(courseId, userEmail);
+    }
 
+    /**
+     * @param courseId
+     * @param userEmail
+     * @return a list of viewable feedback sessions for any user for his course.
+     * @throws EntityDoesNotExistException
+     */
```

Comment:
The way I write it, the old method `getFeedbackSessionsForUserInCourse` calls the skipCheck method after verifying that the course is present. In one way, it can be seen as an afterCheck method. But also it can be directly used, i.e a skipCheck method.
The rationale for writing the skipCheck version is the courseIds passed are obtained from CourseAttributes resulting from query, so checking presence of those courses are redundant.